src/QsFmt/Formatter/SyntaxTree/Statement.fs

```diff
@@ -9,6 +9,37 @@ type SymbolBinding =
     | SymbolDeclaration of SymbolDeclaration
     | SymbolTuple of SymbolBinding Tuple
 
+type QubitSymbolBinding =
+    | QubitSymbolDeclaration of Terminal
+    | QubitSymbolTuple of QubitSymbolBinding Tuple
```

Comment:
Can you remove this type? I think the normal `SymbolBinding` would work fine, right?

Reply:
The difference between `QubitSymbolBinding` and `SymbolBinding` is the type annotation that the `SymbolBinding` has, which is not appropriate for the `QubitSymbolBinding`. You can't, for example, annotate the type of a `use` variable to be a `Qubit` or any other type. It also really bugs me that the `SymbolBinding` type has the annotation at all because that does not match the corresponding rule in the ANTLR grammar. If the `SymbolBinding` type more closely matched that rule by not having a type annotation, I would be more comfortable just using that for the qubit bindings as well.

Reply:
That is not a difference between qubit bindings and let bindings, though, because neither of them can have type annotations. You can create separate types for symbol bindings with types (callable declarations), and symbol bindings without types (let, mutable, use, borrow) if you prefer.

Reply:
I've split out the parameter bindings (those with type annotations) from the symbol bindings (those without annotations) and remove the special qubit symbol bindings. I think this works better than before. Thanks!